src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -32,6 +35,18 @@ namespace ov {
 namespace intel_gna {
 namespace ngraph_util {
 
+using namespace GNAPluginNS::GNALimitations;
+
+static const std::set<ov::element::Type> supported_parameter_types = {
```

Comment:
This is not a but, but keep in mind that in this global variable will have separate copy in each translation unit it will be included.
What is more maybe it would be better to introduce method for checking if parameter type is valid. To hide implementation details.

```
// in header
class SupportedTypes {
public:
bool is_parameter_type_supported(ov::element::Type);
std::set<ov::element::Type> get_supported_constant_types();

bool is_constant_type_supported(ov::element::Type);
std::set<ov::element::Type> get_supported_constant_types (ov::element::Type);

private:
static const std::set<ov::element::Type> supported_parameters_types;
static const std::set<ov::element::Type> supported_constant_types;
};
// implementation in source file
```
less implementation details in headers will allow us later to:
* easier modifications  add  
* reduce compilation time.

Reply:
eventually I would put implementation of two methods ins source 
the same as you did for  `is_op_supported`